.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
 sudo: false
-dist: trusty
+dist: xenial
```

Comment:
Changed Travis CI to use Ubuntu Xenial instead of Trusty given that `rvm` fails to install Ruby 3.* on the latter.